<?xml version="1.0"?>
<launch>
  <node pkg="francor_servo_lx16a" type="francor_servo_lx16a_node" name="francor_servo_lx16a_node" output="screen">
    <param name="servo_xml_cfg"  value="$(find francor_servo_lx16a)/config/servo_default.xml"/>
    <param name="serial_device"  value="/dev/ttyUSB0"/>
    <param name="rate_pos_req"   value="20.0"/>
    <param name="rate_speed_req" value="20.0"/>
    <param name="rate_error_req" value="0.5"/>
    <param name="rate_temp_req"  value="0.5"/>
    <param name="rate_vin_req"   value="1.0"/>
  </node>

  <!-- <node pkg="tf" type="static_transform_publisher" name="parent_to_child_static_tf" args="0 0 0 0 0 0 parent child 100" /> -->

</launch>
